LocalSettings.php

```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
```

Comment:
Therefore you need to remove "checkuser", "checkuser-log", "renameuser" and "usermerge" and then I can merge this PR for changing the founder permissions